db/migrate/20230926021757_create_messages.rb

```diff
@@ -0,0 +1,15 @@
+class CreateMessages < ActiveRecord::Migration[7.0]
+  def up
+    create_table :messages do |t|
+      t.string :content
+      t.timestamps
+    end
+
+    # Insert sample greetings
+    Message.create(content: "Hello, World!")
+    Message.create(content: "Hi there!")
+    Message.create(content: "Greetings!")
+    Message.create(content: "Welcome!")
+    Message.create(content: "Salutations!")
+  end
```

Comment:
- [ ] Seeding the db should be done in the seeds.rb file not directly in the migration file, please make sure to move this to the right place.